core-bundle/src/EventListener/PreviewUrlConvertListener.php

```diff
@@ -55,7 +58,17 @@ public function __invoke(PreviewUrlConvertEvent $event): void
                 return;
             }
 
-            $event->setUrl($page->getPreviewUrl($this->getParams($request)));
+            try {
+                $event->setUrl($page->getPreviewUrl($this->getParams($request)));
+            } catch (RouteParametersException $e) {
+                $route = $e->getRoute();
+
+                if (!$e->getPrevious() instanceof InvalidParameterException || !$route instanceof PageRoute || !$route->getPageModel()->requireItem) {
```

Comment:
This is debatable. 

Swallowing any `RouteParametersException` should definitely be too loose since it catches exceptions like `RouteNotFoundException` as well.

However, swallowing any `InvalidParameterException` could be a more useful default here.
Maybe @aschempp  can share his opinion.

Reply:
I think catching the `InvalidParameterException` is correct here. Not 100% sure though.